tests/recruitment/candidates/searchCandidateSuite/searchCandidate.spec.ts

```diff
@@ -0,0 +1,328 @@
+  // login
+  await loginPage.login(
+    account.adminAccount.username,
+    account.adminAccount.password
+  );
+  // vao recruitmentLink
+  await addRecord(page, recruitmentPage, ValidUser);
+  await recruitmentPage.recruitmentLink.click();
+});
+
+test.describe("Candidate Searching Suite", () => {
+  test("Filter candidates by Job Title", async ({ page }) => {
+    // const vacancy =
+    await recruitmentPage.jobTitle.click();
+    await recruitmentPage.jobTitleName.click();
+    await recruitmentPage.submitButton.click();
+    const fullName = fullNameCombiner(page, ValidUser);
+    await expect(fullName).toBeVisible(); // lý do chỉ check fullname vì nó Unique
+    const firstName = ValidUser.firstName;
+    const candidateName = fullCandidateName(page, ValidUser);
+    await deleteRecord(
+      fullName,
+      firstName,
+      candidateName,
+      page,
+      recruitmentPage
+    );
+  });
+
+  test("Filter candidates by Vacancy", async ({ page }) => {
+    const fullName = fullNameCombiner(page, ValidUser);
+    await recruitmentPage.vacancy.click();
+    await recruitmentPage.vacancyName.click();
+    await recruitmentPage.submitButton.click();
+    await expect(fullName).toBeVisible();
+    const firstName = ValidUser.firstName;
+    const candidateName = fullCandidateName(page, ValidUser);
+    await deleteRecord(
+      fullName,
+      firstName,
+      candidateName,
+      page,
+      recruitmentPage
+    );
+  });
+
+  test("Filter candidates by Status", async ({ page }) => {
+    const fullName = fullNameCombiner(page, ValidUser);
+    await recruitmentPage.status.click();
+    await recruitmentPage.statusName.click();
+    await recruitmentPage.submitButton.click();
+    await expect(fullName).toBeVisible();
+    const firstName = ValidUser.firstName;
+    const candidateName = fullCandidateName(page, ValidUser);
+    await deleteRecord(
+      fullName,
+      firstName,
+      candidateName,
+      page,
+      recruitmentPage
+    );
+  });
+  test("Filter candidates by Candidate Name", async ({ page }) => {
+    const fullName = fullNameCombiner(page, ValidUser);
+    const firstName = ValidUser.firstName;
+    const candidateName = fullCandidateName(page, ValidUser);
+    await recruitmentPage.candidateField.click();
+    await recruitmentPage.candidateField.fill(firstName); //fill first name
+    await candidateName.click();
+    await recruitmentPage.submitButton.click();
+    await expect(fullName).toBeVisible();
+    await deleteRecord(
+      fullName,
+      firstName,
+      candidateName,
+      page,
+      recruitmentPage
+    );
+  });
+
+  test("Filter candidates by Keywords", async ({ page }) => {
+    const fullName = fullNameCombiner(page, ValidUser);
+    await recruitmentPage.keywords.click();
+    const keyWords = ValidUser.keywords;
+    await recruitmentPage.keywords.fill(keyWords);
+    await recruitmentPage.submitButton.click();
+    await page.waitForTimeout(5000);
```

Comment:
chỗ này hạn chế wait hardcode time như này nha, nên xài wait for element to bi visible r để timeout vào đấy